### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by JKL98ISR**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,20 +51,13 @@ pip install deepchecks #--user
...
 ```
-and then either:
+For stable tag:
 ```bash
-pip install .
-```
-or
-```bash
-python setup.py install
+pip install git+https://github.com/deepchecks/deepchecks.git@<tag_name>
```
Comment (ItayGabbay) [inline README.md:60]: Maybe write the latest tag and let's change every release?

File: README.md [hunk 2]
```diff
@@ -51,20 +51,13 @@ pip install deepchecks #--user
...
 
 ### From source
-First clone the repository and then install the package from inside the repository's directory:
+For latest stable version:
 ```bash
-git clone https://github.com/deepchecks/deepchecks.git
-cd deepchecks
-# for installing stable tag version and not the latest commit to main
-git checkout tags/<name_of_latest_tag>
+pip install git+https://github.com/deepchecks/deepchecks.git@0.0.10
```
Comment (matanper) [inline README.md:56]: do we want to have the version in the README? we'll have to update on each release

Comment (JKL98ISR) [reply README.md:56]: I think it's usually like that, I also changed that according to gabbay's comment

Comment (shir22) [reply README.md:56]: I don't think it gives value to add which version
Lets leave it as is

Comment (JKL98ISR) [reply README.md:56]: ok

**Review by shir22 — APPROVED** [review]

**Pull request merged**